packages/core/nuxt-theme-module/theme/components/FiltersSidebar.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <SfSidebar
```

Comment:
This way you'll have issues with lazy loading this component, check how `CartSidebar` has been done

Reply:
Is the additional div enough to avoid lazy loading issues or do you mean adding LazyHydrate component (I added it in Category component when passing FiltersSidebar)?